### PR TITLE
Add bale messenger logo

### PR DESCRIPTION
**Pull request by Thecode764**
**Issue:** closes #

**Popularity metric:**

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by uncenter** [comment]
Please fill out the pull request template that you deleted. I am editing your original comment to put it back.

**Comment by Thecode764** [comment]
@uncenter  wait. what is hex in _data/simple_icons.json?

**Comment by uncenter** [comment]
> @uncenter wait. what is hex in _data/simple_icons.json?

https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#color-guidelines

**Comment by Thecode764** [comment]
@uncenter okay. please check again :)

**Comment by uncenter** [comment]
You still have not provided a preview image or a popularity metric.

File: icons/bale.svg
Comment (uncenter) [inline icons/bale.svg:1]: It doesn't look like you've read any of https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#5-check-the-icon, this has:
- The wrong viewbox.
- Missing `role="img"`.
- Extra non-path elements (`g`) within.
- Extra attributes on the path (`transform`).
- Is not optimized/minified.

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by Thecode764** [comment]
hmmmmmmmmmmmmmmmmmmm. i close this pr

**Pull request closed without merging**